scripts/benchmark_all_pallets.sh

```diff
@@ -14,21 +14,22 @@ echo "WEIGHT_OUTPUT_DIR:  ${WEIGHT_OUTPUT_DIR}"
 mkdir -p "$WEIGHT_OUTPUT_DIR"
 
 pallets=(
-  #"frame_system" \
-  #"pallet_balances" \
-  #"pallet_multisig" \
-  #"pallet_preimage" \
-  #"pallet_proxy" \
-  #"pallet_scheduler" \
+  "frame_system" \
+  "pallet_balances" \
+  "pallet_multisig" \
+  "pallet_preimage" \
+  "pallet_proxy" \
+  "pallet_scheduler" \
   "pallet_teerex" \
   "pallet_sidechain" \
   "pallet_enclave_bridge" \
-  #"pallet_claims" \
-  #"pallet_timestamp" \
-  #"pallet_treasury" \
-  #"pallet_vesting" \
-  #"pallet_utility" \
+  "pallet_claims" \
+  "pallet_timestamp" \
+  "pallet_treasury" \
+  "pallet_vesting" \
+  "pallet_utility" \
   "pallet_teeracle" \
+  "pallet_teerdays" \
```

Comment:
We have changed the code in the parachain to read all pallets from the runtime directly: https://github.com/integritee-network/parachain/blob/master/scripts/benchmark_all_pallets.sh

Reply:
thx. I do, however, like the simplicity to benchmark a single pallet like here - at least for the solochain node, where we may want to test and iterate benchmarks